android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -186,10 +186,15 @@ class RNPlayerView(
      * Cleans up the resources and listeners produced by this view.
      */
     fun dispose() {
-        viewEventRelay.eventEmitter = null
-        playerEventRelay.eventEmitter = null
-        playerView?.removeOnLayoutChangeListener(this)
-        playerView?.onDestroy()
+        _playerView?.run {
```

Comment:
- I would prefer using a scope with a named parameter, avoiding the need to use `@RNPlayerView`. 
- Also do we not need the eventEmitter handling anymore?

Reply:
I don't think that other scope functions are actually applicable. Best matching would be `with` actually.. but then I have to explicitly take care of the nullability and we still have to specify which `this` we are using.
(`let` and `also` do not make sense from a wording perspective)

`eventEmitter` is handled by detatching (setting to `null`) the `_playerView`

https://github.com/bitmovin/bitmovin-player-react-native/pull/323/commits/ebd580ae7af786bdb422ae054258ef0d33b0aeec
https://github.com/bitmovin/bitmovin-player-react-native/pull/323/commits/beb7519a6a67603c35b986bffd8d2b04fa93e66e

Reply:
Very nice!